doc/enterprise/auth.md

```diff
@@ -0,0 +1,102 @@
+
+(2) Please paste the token you receive from GitHub here:
+
+```
+
+When visiting this link in a browser, you will be presented with an option to "Authorize Pachyderm" (assuming that you haven't authorized Pachyderm via GitHub previously). Once you authorize Pachyderm, you will be presented with a Pachyderm user token:
+
+![alt tag](auth.png)
+
+Copy and paste this token back into the terminal, as requested by `pachctl`, and press enter.  You are now logged in to Pachyderm!  
+
+## Managing and updating user access
+
+Let's suppose that we create a repository call `myrepo` with the user `dwhitena`.  Because, the user `dwhitena` created this repository, `dwhitena` will have full read/write access to the repo.  This can be confirmed via the `pachctl auth check ...` command:
+
+```
+$ pachctl auth check owner myrepo
+true
+$ pachctl auth check reader myrepo
+true
+$ pachctl auth check writer myrepo
+true
+```
+
+Or via the `pachctl auth get ...` command:
+
+```
+$ pachctl auth get dwhitena myrepo
+OWNER
+```
+
+An OWNER of `myrepo` or a cluster admin can then set other user's scope of access to the repo.  This can be done via the `pachctl auth set ...` command.  For example, to give the GitHub user `JoeyZwicker` READER (but not WRITER or OWNER) access to `myrepo` we execute:
+
+```
+$ pachctl auth set JoeyZwicker reader myrepo
+```
+
+This updated scope can then be verified with `pachctl auth get ...`:
+
+```
+$ pachctl auth get JoeyZwicker myrepo                        
+READER
+```
+
+## Behavior of pipelines as related to access control
+
+In Pachyderm, you don't explicitly set the scope of access for users on pipelines.  Rather, pipelines infer access from the repositories that are input to the pipeline, as follows:
+
+- An OWNER, WRITER, or READER of a repo may subscribe a pipeline to that repo.
+- When a user subscribes a pipeline to a repo, they will be set as an OWNER of that pipeline's output repo.
+- The initial OWNER of a pipeline's output repo (or an admin) needs to set the scope of access for other users to that output repo.  
+
+## Activation code expiration and de-activation
+
+When an enterprise activation code expires, an auth-activated Pachyderm cluster goes into an "admin only" state.  In this state, only admins will have access to data that is in Pachyderm.  This safety measure keeps sensitive data protected, even when an enterprise subscription becomes stale. As soon as the enterprise activation code is updated (via the dashboard or via `pachctl enterprise activate ...`), the Pachyderm cluster will return to it's previous state.
+
+When access controls are de-activated on a Pachyderm cluster via `pachctl auth deactivate`, the cluster returns to being a non-access controlled Pachyderm cluster.  That is, all ACLs are deleted and the cluster returns to being a blank slate in regards to access control.
```

Comment:
Change "blank slate" or define it -- everyone will have access to all repos. 

If Auth is deactivated, that also removes all notions of users from the system, right? So there isn't even a way to log in anymore, etc? 

Lastly, it says that "all ACLs are deleted", but this is a really big deal (especially since ever ACL has to be manually added in our current MVP auth release) so perhaps break into a bullet list or something that emphasizes it even stronger.